src/Parser/Shaman/Elemental/Modules/ShamanCore/FlameShock.js

```diff
@@ -76,7 +38,7 @@ class FlameShock extends Analyzer {
         value={`${formatPercentage(this.uptime)} %`}
         label="Uptime"
         tooltip={
```

Comment:
When not joining strings you can just provide a regular (double quoted) string instead of a template field.